orangecontrib/text/widgets/owcorpustonetwork.py

```diff
@@ -0,0 +1,197 @@
+from typing import Tuple, Any
+
+from Orange.widgets.widget import OWWidget, Input, Output, Msg
+from Orange.widgets.settings import Setting
+from Orange.widgets.gui import widgetBox, comboBox, spin, button
+from Orange.widgets.utils.concurrent import ConcurrentWidgetMixin, TaskState
+from Orange.data import Table
+
+from orangecontrib.network import Network
```

Comment:
This would make that widget does not show up when someone does not have a Network addon installed. Is it ok? @ajdapretnar do we have any similar case of widget that needs the other addon anywhere else? what is the practice in this case? Is it ok that widget just does not shows up or should it shows up and show an error - since it needs a network addon.